packages/server/graphql/mutations/helpers/inviteToTeamHelper.ts

```diff
@@ -78,11 +37,30 @@ const inviteToTeamHelper = async (
   const operationId = dataLoader.share()
   const subOptions = {mutatorId, operationId}
 
-  const trustScore = await getInviteTrustScore(viewerId, teamId, invitees, dataLoader)
-  if (trustScore < 0.15)
-    return {error: {message: 'Cannot invite by email. Try using invite link'}, invitees: []}
+  const untrustedDomains = ['tempmail.cn', 'qq.com']
+  const filteredInvitees = invitees.filter(
+    (invitee) => !untrustedDomains.includes(getDomainFromEmail(invitee).toLowerCase())
+  )
+  const mailManager = new MailManagerMailgun()
```

Comment:
-1 We should not hardcode it to this as we don't want to hit the network in tests and local development if not necessary.

Reply:
What do you suggest instead? Check if it's production and only use `MailManagerMailgun` if it is? Locally & in tests, skip the validation?

Reply:
Just following up on this message and [this one](https://github.com/ParabolInc/parabol/pull/8707#discussion_r1326020199) in case you haven't seen them @Dschoordsch

Reply:
We have [`getMailManager`](https://github.com/ParabolInc/parabol/blob/434f85f016c8d7e7896c7578b353fdc95deff2b3/packages/server/email/getMailManager.ts#L17) which returns the correct manager based on environment. I think you should extend the `MailManager` interface with the new `validateEmail(...)` function. You could either make it optional and only call it if it's present, or you could make a dummy implementation in the others which returns true based on some basic filters or similar.

Reply:
Thanks, that makes sense. Updated 👍